app/assets/javascripts/admin/enterprise_groups/controllers/side_menu_controller.js.coffee

```diff
@@ -0,0 +1,17 @@
+angular.module("admin.enterprise_groups")
+  .controller "sideMenuCtrl", ($scope, SideMenu) ->
+    $scope.menu = SideMenu
+    $scope.select = SideMenu.select
+
+    $scope.menu.setItems [
+      { name: 'Primary Details', icon_class: "icon-user" }
+      { name: 'About', icon_class: "icon-pencil" }
+      { name: 'Images', icon_class: "icon-picture" }
+      { name: 'Contact', icon_class: "icon-phone" }
+      { name: 'Web', icon_class: "icon-globe" }
+    ]
+
+    $scope.select(0)
+
+    $scope.showItem = (item) ->
```

Comment:
Given the hiccup you ran into with this before, perhaps we could change SideMenu to automatically show items if this method isn't defined?